web_datetime_picker_default_time/static/src/js/datetimepicker_service.esm.js

```diff
@@ -0,0 +1,295 @@
+import {patch} from "@web/core/utils/patch";
+import {
+    areDatesEqual,
+    formatDate,
+    formatDateTime,
+    parseDate,
+    parseDateTime,
+} from "@web/core/l10n/dates";
+import {localization} from "@web/core/l10n/localization";
+import {ensureArray, zip, zipWith} from "@web/core/utils/arrays";
+import {deepCopy} from "@web/core/utils/objects";
+import {datetimePickerService} from "@web/core/datetime/datetimepicker_service";
+
+const FOCUS_CLASSNAME = "text-primary";
+
+const formatters = {
+    date: formatDate,
+    datetime: formatDateTime,
+};
+
+const listenedElements = new WeakSet();
+
+// OVERRIDE:
```

Comment:
See my other comment about trying to keep here only the methods that are actually overriden